crowdsec-docs/unversioned/user_guides/plugins/whm.md

```diff
@@ -0,0 +1,152 @@
+
+If you don't want the script to install the WHM collection, you can use the `--only-plugin` option:
+
+```
+sudo sh crowdsec.sh install --only-plugin
+```
+
+
+### Navigate to the plugin in WHM
+
+
+CrowdSec should appear in the sidebar in the **Plugins** section.  
+Easily find it by filtering your sidebar by starting to type ***"crowdsec"***
+
+
+## Post-installation checks
+
+Once installation is complete, you can perform a few post-installation tasks to ensure that everything works as expected.
+
+
+### Check the CrowdSec service status
+
+At the top of all CrowdSec plugin pages, you can see the status of the CrowdSec service: if there's a green tick, the service is running. 
+
+![Service status](img/whm-service-status.png)
+
+If not, [check the troubleshooting section](#crowdsec-is-not-running).
+
+### Check the metrics
+
+Browse to the metrics tab and ensure that The CrowdSec security engine 
+
+![Metrics](img/whm-metrics.png)
+
+If not, [check the troubleshooting section](#changing-port-configuration).
+
+### Check the default acquisition files
+
+The plugin comes with a set of default acquisition configuration files tailored for WHM typical logs directories.  
+Those files are created in `etc/crowdsec/acquis.d/`, each file allows reading the logs for the various services you want to protect. 
+Note that the main acquisition file is `/etc/crowdsec/acquis.yaml`, but all additional acquisitions should be placed in `/etc/crowdsec/acquis.d`.  
+
+The **acquisition tab** of the plugin will list the various acquisition configuration files and a summary of their contents.
+This section is for a somewhat advanced usage of CrowdSec. If you installed WHM with the default configuration the default acquisitions should be sufficient.
+
+What is being parsed can be seen in the **metrics tab** of the plugin.  
+The principal thing you should be concerned about is that your web server logs are properly parsed which for the default cases means that the apache2 logs are properly read.
+
+Some acquisitions may display a warning (⚠)sign next to the "logs filenames" that indicates that "nothing was parsed from that path" since the last CrowdSec service restart.  
+Depending on the activity on your server it might be normal, you can check the lines parsed in the **metrics > acquisition** section.  
+Usually, you should see activity within a few minutes.
+
+![Acquisition not read](img/whm-acquisition-not-read.png)
+
+### Enroll your engine in CrowdSec's Console
+
+CrowdSec's console can provide additional insights into what's happening on your park of security engines, along with additinonal features allowing you, among other things, to feed your security engine additional blocklists to protect your perimeter (if you want for example to block botnets of tor nodes from accessing your server).  
```

Comment:
```
CrowdSec's console offers deeper insights into the activities within your network of security engines. It also includes extra functionalities that enable you to enhance your security engine's capabilities, such as integrating additional blocklists. This feature is particularly useful if you wish to prevent botnets or Tor nodes from accessing your server, among other potential security enhancements.
```

Reply:
The wording is abit off, here a reworded statement to make it concise